shared/graphql/schema.graphql

```diff
@@ -141,6 +146,7 @@ type BucketConfig {
   associatedPolicies: [PolicyBucketPermission!]! @admin
   associatedRoles: [RoleBucketPermission!]! @admin
   collaborators: [CollaboratorBucketConnection!]!
+  tabulatorTables: [TabulatorTable!]! @admin
```

Comment:
i thought you wanted to make this available to non-admins as well

Reply:
yeah, just forgot to update it from enterprise